pkg/apis/serving/v1alpha1/domainmapping_types.go

```diff
@@ -69,6 +69,7 @@ type DomainMappingList struct {
 // DomainMappingSpec describes the DomainMapping the user wishes to exist.
 type DomainMappingSpec struct {
 	// Ref points to an Addressable.
+	// Currently, Ref must be a KSvc.
```

Comment:
Why?

Reply:
I see the other comment, in practice I think it works with more than that, but this should be pretty easy to resolve more broadly (assuming the addressable doesn't use the path part, and implements our dataplane contract).

Comment seems fine for now, we should just be sure we clean it up 😅

Reply:
Yeah, I guess I'm a little conservative with the comment here - undercommit and overdeliver etc ;) Supporting arbitrary addressables is in the todo list in the tracking issue, though.